tests/datasets/configuration/test_entity_loader.py

```diff
@@ -105,6 +105,7 @@ def test_entity_loader_no_custom_validators(self) -> None:
         entity_validators = set(pluggable_entity.get_validators())
         assert len(entity_validators) == len(pluggable_entity._get_builtin_validators())
 
+    @pytest.mark.skip(reason="Dataset no longer exists")
```

Comment:
The error is `KeyError: StorageKey.GROUPASSIGNEES`. I think that's the reason why a lot of tests needed to be skipped. If we can solve this, it might solve a lot of problems.

Reply:
OK I think I found that problem. I changed a fixture. I'm not sure which tests you commented out because of that.

Reply:
Nice!! I can go through and re-audit them later to remove unnecesary skips

Reply:
Did you solve it by changing the storage key to "discover"?

Reply:
Yeah that was my solution. This is a fixture entity, so I don't think it really matters.

Ideally, we would have a full set of fixtures (entity, storage, storage keys, migrations) that are specific to tests. Then we can reference one of those.

Reply:
i took a closer look at this. Rewriting all the tests to use discover is still quite a bit of work. Then we also have the problem that we shifted the problem - discover should probably not exist anyway and was always intended to be temporary. I have no issue with this approach for now, but don't have time to update all these tests right this now.

Reply:
OK well I think this might sit until project week. Making the tests independent of production configuration seems like a great project week thing though.